config/vendor_api/v1.1.yml

```diff
@@ -1179,19 +1176,24 @@ components:
       type: object
       additionalProperties: false
       required:
+      - id
       - message
-      - date
+      - updated_at
```

Comment:
Wondering whether we should publish `created_at` instead of `updated_at`. Notes are special in that we currently don't allow edits. If the content of a note is tweaked via the rails console, we probably still want to show them in `created_at` order.

Reply:
I agree

Reply:
We've added `created_at` to both notes and interviews, but have also kept `updated_at`.

Reply:
👌